### PR TITLE
Fix initial load for free schema + TESTS

### PR DESCRIPTION
**Pull request by aliciakw**

### CODE REVIEW

File: tests/unit/services/file.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  if (window.performance && typeof window.performance.now === 'function') {
+    d += performance.now(); // use high-precision timer if available
+  }
+  const uuid = 'xxxxxx'.replace(/[xy]/g, (c) => {
+    const r = (d + Math.random() * 16) % 16 | 0;
+    d = Math.floor(d / 16);
+    return (c === 'x' ? r : (r & 0x3 | 0x8)).toString(16);
+  });
+  return uuid;
+};
```
Comment (hhff) [inline tests/unit/services/file.js]: Is this file necessary?

**Review by hhff — APPROVED** [review]
LGTM outside of extra file. Let's setup CircleCI on Monday!

File: tests/unit/services/file.js (hunk 1 above)
Comment (aliciakw) [reply tests/unit/services/file.js]: oh whoops was testing something

**Review by limsohee1002 — APPROVED** [review]

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
+version 2
+
+defaults: &defaults
+  working_directory: ~/contentful-fragment
```
Comment (hhff) [inline .circleci/config.yml]: u can snip this

File: .circleci/config.yml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  docker:
+    - image: circleci/node:8
+
+jobs:
+  install-dependencies:
+    <<: *defaults
+    steps:
+      - checkout
+      - attach_workspace:
+        at: ~/contentful-fragment
```
Comment (hhff) [inline .circleci/config.yml]: and this!

File: .circleci/config.yml [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+        - v1-dependencies-{{ checksum "yarn.lock" }}
+        - v1-dependencies-
+      - run: yarn install
+      - save_cache:
+        paths:
+          - node_modules
+        key: v1-dependencies-{{ checksum "yarn.lock" }}
+      - persist_to_workspace:
+        root: .
+        paths: .
```
Comment (hhff) [inline .circleci/config.yml]: and this

File: .circleci/config.yml [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+  run-tests:
+    <<: *defaults
+    steps:
+      - attach_workspace:
+        at: ~/contentful-fragment
```
Comment (hhff) [inline .circleci/config.yml]: and dis

File: .circleci/config.yml [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+  install-dependencies:
+    <<: *defaults
+    steps:
+      - checkout
+      - attach_workspace:
+        at: ~/contentful-fragment
+      - restore_cache:
+        keys:
+        - v1-dependencies-{{ checksum "yarn.lock" }}
+        - v1-dependencies-
```
Comment (hhff) [inline .circleci/config.yml]: u can lose this cause we shouldn't have a non-specific cache key

**Review by hhff — CHANGES_REQUESTED** [review]

**Review by joshiefishbein — APPROVED** [review]

**Pull request merged**